portuguese.json

```diff
@@ -0,0 +1,119 @@
+    "command.tpto.declined.receiver": "Recusou o pedido de {%sender} para que eles se teletransportassem para você.",
+    "command.tpto.declined.sender": "{%receiver} recusou o seu pedido de se teletransportar para eles.",
+    "command.tpto.noperm": "Não tem autorização para efetuar a teletransportação para outros jogadores. Adquira o passe §l§aEMERALD§r §§mor §l§bLEGEND§r §§mrank em  §§angmc.co/store §§mno site!",
+    "command.tpto.norequest": "Aquele jogador não enviou um pedido de teletransporte para você.",
+    "command.tpto.receive": "{%sender} pediu para se teletransportar para você! Use §§a/tpto accept {%0} para aceitar o pedido.",
+    "command.tpto.send": "O seu pedido de teletransporte para {%receiver} foi enviado.",
+    "command.vanish.disabled": "Agora você está visível!",
+    "command.vanish.enabled": "Agora você está invisivel",
+    "command.vote.slowdown": "Calma, já estamos verificando os votos para você!",
+    "db.error": "Ocorreu um erro inesperado na conexão com o banco de dados. Por favor, tente novamente mais tarde.",
+    "formhandler.reporter.notreported": "Esse jogador não pode ser reportado.",
+    "formhandler.reporter.reported": "Você reportou {%reported} por {%reason}",
+    "formhandler.reporter.staffmessage": "{%reported} foi reportado {%reporter} por {%reason}",
+    "formhandler.settings.disabled": "Desativado",
+    "formhandler.settings.enabled": "Ativado",
+    "formhandler.settings.fly": "{%enabledOrDisabled} voar no lobby.",
+    "formhandler.settings.fly.hidingrank": "Você não pode voar enquanto estiver usando um nome falso ou a esconder a sua posição de rank.",
+    "formhandler.settings.friends": "Agora vocês {%enabledOrDisabled} São amigos.",
+    "formhandler.settings.guildtag": "Agora é {%enabledOrDisabled} your guild tag.",
+    "formhandler.settings.hideplayers": "You are now {%enabledOrDisabled} players.",
+    "formhandler.settings.hiding": "escondido",
+    "formhandler.settings.knockback": "{%enabledOrDisabled} knockback no lobby.",
+    "formhandler.settings.announcements": "{%enabledOrDisabled} anúncios do servidor.",
+    "formhandler.settings.nick.enabled": "O sua nametag foi alterado para {%nametag}!",
+    "formhandler.settings.nick.reset": "Reajustando sua nametag.",
+    "formhandler.settings.pet.removed": "Pet removido.",
+    "formhandler.settings.pet.selected": "O seu pet de estimação foi mudado para um {%pet}.",
+    "formhandler.settings.showing": "exibindo",
+    "formhandler.switcher.doesntexist": "Uma conta com esse nome de usuário não existe! Por favor tente novamente.",
+    "forms.party.invite": "Convidar mais jogadores:",
+    "forms.reporter": "Reportar",
+    "forms.settings.fly": "Alternar voo",
+    "forms.settings.friends": "Permitir pedidos de amigos",
+    "forms.settings.hideguildtag": "Ocultar tag de guilda",
+    "forms.settings.hideplayers": "Ocultar Jogadores",
```

Comment:
"Jogadores" shouldn't be capitalized here.

Reply:
Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).